issuer_controller/src/app.py

```diff
@@ -154,7 +165,7 @@ def agent_callback(topic):
         end_time = time.perf_counter()
         issuer.log_timing_method(method, start_time, end_time, False)
         issuer.log_timing_event(method, message, start_time, end_time, False)
-        abort(400, {'message': 'Invalid topic: ' + topic})
+        abort(400, {"message": "Invalid topic: " + topic})
```

Comment:
Does your editor automatically change single- to double-quotes?

Reply:
It does when I format the code (not sure whether this is because of the editor or `black` config though).